rollup.config.js

```diff
@@ -0,0 +1,41 @@
+import babel from "rollup-plugin-babel";
+import commonjs from "rollup-plugin-commonjs";
+import localResolve from "rollup-plugin-local-resolve";
+import resolve from "rollup-plugin-node-resolve";
+import peerDepsExternal from "rollup-plugin-peer-deps-external";
+import postcss from "rollup-plugin-postcss";
+
+import pkg from "./package.json";
+
+export default {
+  input: "src/overridable.js",
+  output: [
+    {
+      file: pkg.main,
+      format: "cjs",
+      sourcemap: true,
+    },
+    {
+      file: pkg.module,
+      format: "esm",
+      exports: "named",
+      sourcemap: true,
+    },
+  ],
+  plugins: [
+    peerDepsExternal(),
+    postcss({
+      plugins: [],
+      minimize: true,
+      sourceMap: "inline",
+    }),
```

Comment:
probably not needed since no css is involved here

Reply:
Yes, sorry, I will clean up.